core/domain/classifier_services.py

```diff
@@ -127,6 +127,75 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+def handle_classifier_training_job_creation(exploration, state_names):
```

Comment:
I think, in this case, it's still fine to have just one function, otherwise there's unnecessary indirection.